content/en/docs/setup/install/multicluster/common.sh

```diff
@@ -101,6 +101,9 @@ function cleanup_cluster2
 # between CLUSTER1 and CLUSTER2.
 function verify_load_balancing
 {
+  # Verify istiod is synced
+  snip_verify_multicluster_1
```

Comment:
simply calling verify_multicluster wont be enough.. you will have to compare the output of the command to the expected output which is there in snip_verify_multicluster_1_out  . Take a look at the usages of _verify_like() in other similar tests